### ISSUE TITLE
Bad event used in navigation leads to scrolling embedded player when zooming the model

### ISSUE THREAD

**Issue by X-Ryl669** [kovacsv/Online3DViewer#68]
That's because the code is using deprecated (mozilla only) "DOMMouseScroll" and "mousewheel" event while it should use "wheel" event instead, see [here](https://developer.mozilla.org/en-US/docs/Web/API/Element/DOMMouseScroll_event):

With this patch, it works correctly:
```diff
diff --git a/source/viewer/navigation.js b/source/viewer/navigation.js
index 9848490..8ff15d6 100644
--- a/source/viewer/navigation.js
+++ b/source/viewer/navigation.js
@@ -249,8 +249,7 @@ OV.Navigation = class

                if (this.canvas.addEventListener) {
                        this.canvas.addEventListener ('mousedown', this.OnMouseDown.bind (this));
-                       this.canvas.addEventListener ('DOMMouseScroll', this.OnMouseWheel.bind (this));
-                       this.canvas.addEventListener ('mousewheel', this.OnMouseWheel.bind (this));
+                       this.canvas.addEventListener ('wheel', this.OnMouseWheel.bind (this));
                        this.canvas.addEventListener ('touchstart', this.OnTouchStart.bind (this));
                        this.canvas.addEventListener ('touchmove', this.OnTouchMove.bind (this));
                        this.canvas.addEventListener ('touchend', this.OnTouchEnd.bind (this));
@@ -487,6 +486,8 @@ OV.Navigation = class
                        delta = -params.detail;
                } else if (params.wheelDelta) {
                        delta = params.wheelDelta / 40;
+               } else if (params.deltaY) {
+                       delta = -params.deltaY / 40;
                }

                let ratio = 0.1;
```

**Comment by kovacsv** [comment]
Thanks you for the tip! Could you please open a pull request with the fix?

**Comment by X-Ryl669** [comment]
I've 2 opened PR right now and I'm maintaining a merged branch (not submitted yet since it's what the repository should look like once the 2 PR will be merged) so I'm already late from the dev branch. I'll open a PR once the other are merged to avoid maintaining so many branches.

**Comment by kovacsv** [comment]
Thanks for pointing this out, I've added the fix on the dev branch.

**Comment by kovacsv** [comment]
Released in 0.7.10 version.
